ncurses.yaml

```
@@ -1,7 +1,7 @@
 package:
   name: ncurses
-  version: 6.5_p20241006
-  epoch: 4
+  version: 6.5_p20241228
+  epoch: 5
```

```suggestion
  epoch: 0
```

If you bump the version you don't have to bump the epoch